### PR TITLE
Fixing tests: RunCronOverwrite and CronOverwrite

### PR DESCRIPTION
**Pull request by acohen4**

### CODE REVIEW

**Comment by Tryneus** [comment]
LGTM

I'll take care of that TODO in my upcoming PR for #6366

**Review by Tryneus — APPROVED** [review]

File: src/server/pps/server/monitor.go [hunk 1]
```diff
@@ -456,7 +456,7 @@ func (m *ppsMaster) makeCronCommits(ctx context.Context, in *pps.Input) error {
 		}
 		if in.Cron.Overwrite {
 			// get rid of any files, so the new file "overwrites" previous runs
-			err = pachClient.DeleteFile(client.NewCommit(in.Cron.Repo, "master", ""), "")
+			err = pachClient.DeleteFile(client.NewCommit(in.Cron.Repo, "master", ""), "/")
```
Comment (brycemcanally) [inline src/server/pps/server/monitor.go:459]: :+1:

Comment (brycemcanally) [reply src/server/pps/server/monitor.go:459]: I am thinking about making this more structured so we don't need to depend on the trailing slash.

File: src/server/pachyderm_test.go [hunk 2]
```diff
@@ -6162,26 +6157,34 @@ func TestCronPipeline(t *testing.T) {
...
 
-				// subscribe to the pipeline1 cron repo and wait for inputs
-				repo = fmt.Sprintf("%s_%s", pipeline7, "time")
 				ctx, cancel = context.WithTimeout(context.Background(), time.Second*120)
 				defer cancel() //cleanup resources
 				// We expect to see four commits, despite the schedule being every minute, and the timeout 120 seconds
 				// We expect each of the commits to have just a single file in this case
 				// We check four so that we can make sure the scheduled cron is not messed up by the run crons
-				countBreakFunc := newCountBreakFunc(4)
+				countBreakFunc := newCountBreakFunc(5) // Right now we receive an additional Alias Commit. Will change this back to 4 as soon as we can filter subscribeCommit by commitType
```
Comment (brycemcanally) [inline src/server/pachyderm_test.go:6165]: Probably should add `TODO:` to the comment.

Comment (brycemcanally) [reply src/server/pachyderm_test.go:6165]: Yeah, this logic should probably get simplified when we can filter out the alias commits.

**Review by brycemcanally — APPROVED** [review]
lgtm, just a minor change.

**Pull request merged**